pyramid_caching/serializers.py

```
@@ -49,32 +50,24 @@ def register_serialization_adapter(self, object_class, adapter_factory):
     def dumps(self, obj, adapter=None):
         if adapter is None:
             adapter = self.registry.queryAdapter(obj, ISerializationAdapter)
```

We should raise an exception when no adapter is found, shouldn't we?